snf-ui/app/index.html

```diff
@@ -20,6 +20,7 @@
   </head>
   <body class="{{content-for 'body-cls'}}">
     {{content-for 'body'}}
+    <script src="assets/ace/ace.js"></script>
```

Comment:
Use mediaURL here as the file fails to be served when app is installed from debian packages.

> <script src="{{content-for 'mediaURL'}}assets/....